Add privatebin extension

sedlacek-martin opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

sedlacek-martin commented:
Not stale, still ready for review

pernielsentikaer reviewed (changes requested):
Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

sedlacek-martin commented:
I have addressed all the comments from the last review, so can you please have another look, @greptileai?

sedlacek-martin commented on extensions/privatebin/src/create.tsx:
```diff
@@ -0,0 +1,143 @@
+    }
+  };
+
+  useEffect(() => {
+    if (values.format === "plaintext" && isMarkdown(values.pasteData)) {
+      setValue("format", "markdown");
+      showToast({
+        style: Toast.Style.Success,
+        title: "Markdown detected. Changed the format of the paste",
+      });
+    }
+  }, [values.pasteData]);
```
This is, in my opinion, ensured by the TS type. So, there is no need to validate it here.

sedlacek-martin commented:
Thank you for your feedback and the time you took to review my extension!

I have addressed all the concerns, and I think my extension is ready to be reviewed again, @pernielsentikaer.
